File: Dev_Lab Unofficial/Python/Lv Up.py
a = list(map(str,input().split()))
exp = int(a[1])
maxexp = 100
lv = 1
hp = 100
atk = 10

while exp >= maxexp:
  a = exp//maxexp
  exp -= maxexp
  maxexp = 100+100*lv
  lv +=1
hp = hp*lv
atk = atk*lv
print('Lv : {}'.format(lv))
print('Exp : {}/{}'.format(exp,maxexp))
print('ATK : {}'.format(atk))
print('HP : {}'.format(hp))

File: Dev_Lab Unofficial/C++/Lv Up.cpp
#include <iostream>
#include <string>
using namespace std;


int main()
{
	string a;
  int exp;
  cin >> a >> exp;
  int maxexp,atk,hp,lv;
  lv = 1;
  maxexp = lv*100;
  atk = lv*10;
  hp = lv*100;
  
  
  while(exp >= maxexp){
    exp -= maxexp;
    lv +=1;
    maxexp = lv*100;
  	atk = lv*10;
  	hp = lv*100;
  }
  cout << "Lv : " <<lv << endl;
  cout << "Exp : " <<exp << "/" << maxexp << endl;
  cout << "ATK : " <<atk << endl;
  cout << "HP : " <<hp << endl;
	return 0;
}